src/ParseQuery.js

```diff
@@ -1422,6 +1435,24 @@ class ParseQuery {
     query._andQuery(queries);
     return query;
   }
+
+  /**
+   * Constructs a Parse.Query that is the NOR of the passed in queries.  For
+   * example:
+   * <pre>const compoundQuery = Parse.Query.nor(query1, query2, query3);</pre>
+   *
+   * will create a compoundQuery that is a nor of the query1, query2, and
+   * query3.
+   * @param {...Parse.Query} var_args The list of queries to AND.
+   * @static
+   * @return {Parse.Query} The query that is the AND of the passed in queries.
+   */
```

Comment:
Docs are wrong

Reply:
Is there a jsdocs checker / linter we can add?

Reply:
I mean the text in the docs ;) it says AND as it should says NOR. :)

Reply:
The PHP SDK has a document checker. In this case the @param is wrong also

Reply:
Not sure about that, you should look at them with `npm run docs`.

My point was on the whole documentation that it didn’t match what the function was doing